### DISCUSSION TITLE
Page Description syntax / purpose?

### DISCUSSION THREAD

**Discussion by iamkyoumei** [ethercreative/seo#375] [Q&A]
Just wondering what this area is for and if possible, how could I get this to display the first x number of characters for an entries' description?

![image](https://user-images.githubusercontent.com/42029255/134319341-47896aba-9f9f-4a22-bdc0-c10f4a95768d.png)

I couldn't find any documentation regarding this particular entry, so I am not sure if I am just using it wrong as it doesn't seem to change anything in my templates.

**Comment by alexjcollins** [comment]
The page description allows you to define a default description wherever the field type is used when creating a new entry. Existing entries won't inherit the content you add in the settings.

If you wanted to dynamically load a description you would need to first define the custom field on the entry you wanted to pull from. For example, you could have a field on an entry called `{{ entry.text }}`. If you wanted to load the first x characters from that field into the SEO description (`<meta name="description" ...`), you could have something like `{text|slice(0, 50)}` using [Twig slice](https://twig.symfony.com/doc/3.x/filters/slice.html) to limit the characters.

**Reply by iamkyoumei** [reply]
Hi Alex,

So I've tried doing as you have suggested, but this still doesn't appear to be picking up anything in the page description when I create new entries, unlike the title.

I've tried putting in { title } and {{ entry.title }}, but the page description snippet still does not appear to update like the title field does when creating an entry.

The page description does come in prefilled (unlike the title, which is blank) and I actually don't know where it is pulling this title from as I don't recall setting up this 'default' page description.

Would you have any further advice as to what I should do here?

Thank you!
